fitbenchmarking/fitbenchmarking
# Add date and time to fitbenchmarking log #1090

#### Description of Work

Fixes #1090


#### Testing Instructions

1. Check if the fitbenchmarking.log includes the date and time 
2.
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
